services/secondary-storage/controller/src/main/java/org/apache/cloudstack/secondarystorage/SecondaryStorageManagerImpl.java

```diff
@@ -744,9 +743,6 @@ private void allocCapacity(long dataCenterId, SecondaryStorageVm.Role role) {
                 if (_allocLock.lock(ACQUIRE_GLOBAL_LOCK_TIMEOUT_FOR_SYNC)) {
                     try {
                         secStorageVm = startNew(dataCenterId, role);
-                        for (UploadVO upload : _uploadDao.listAll()) {
```

Comment:
@ravening would that cause a regression, to not expunge uploaded volumes?

Reply:
@ravening can you answer this question, please?

Reply:
@rhtyd @DaanHoogland I tested by uploading volumes from local as well as from url but `upload` tables were still empty. So looks like uploaded volume/template entries are not stored in this table.